models/cartDao.js

```diff
@@ -0,0 +1,136 @@
+// HTTP GET 요청에 응답한 HTTP RESPONSE를 위한 함수
+// REQUEST.BODY에 내용을 담는다.
+const { appDataSource } = require("../entity/utils")
+
+
+const makeCartPending = async (userId, cartId) => { // 카트를 결제 대기 상태(pending)로 바꿔 줍니다
+        try {
+                return await appDataSource.query(
+                    `
+                    UPDATE cart 
+                    SET 
+                    status = "PENDING" 
+                    WHERE 
+                    id = '${cartId}' 
+                    AND 
+                    status = "NONE" 
+                    AND 
+                    users_id = '${userId}'
+                    `
+                )
+        } catch (e) {
+                console.error(e);
+        }
+}
+
```

Comment:
```suggestion
const updateCartStatus = async (cartId, status) => { // 카트를 결제 대기 상태(pending)로 바꿔 줍니다
        try {
                return await appDataSource.query(
                    `
                    UPDATE cart 
                    SET 
                    status = "${status}" 
                    WHERE 
                    id = '${cartId}' 
                    `
                )
        } catch (e) {
                console.error(e);
        }
}

```
1. 카트를 펜딩으로만 바꾸는 것이 아니라, 상태를 update 할 수 있는 조금더 일반적인 기능들을 수행할 수 있도록 추상화를 해서, 확장성 높은 함수를 만드는 것이 좋을 것 같습니다.
2. status는 service layer에서 내려주면 더 좋을 것 같아요!